schema/formats.go

```diff
@@ -46,22 +47,53 @@ func (f macFormatChecker) IsFormat(input string) bool {
 	return match
 }
 
+func matchYangIPv4AddressPattern(input string) bool {
+	res, err := regexp.MatchString(`^(([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])(%[\p{N}\p{L}]+)?`, input)
```

Comment:
i suggest https://golang.org/pkg/regexp/#MustCompile for small efficiency gain

Reply:
Done